### DISCUSSION TITLE
Has anyone tried this plugin with obs 30.0?

### DISCUSSION THREAD

**Discussion by mdamaged** [sorayuki/obs-multi-rtmp#363] [Q&A] [answered]
Has anyone tried this plugin with obs 30.0? I do not want to update obs otherwise. Thanks!

**Comment by sorayuki** [comment, marked as answer]
Release targeted OBS 30.0.0 is now published.

**Reply by SabreRunner** [reply]
Is there some guide to setup properly and covering all OBS 30's new features. I want to make sure I don't mess it up.
